src/gg.regression.unity.bots/Runtime/Scripts/RGOverlay/RGDeleteSequence.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace RegressionGames
+{
+    /**
+     * <summary>
+     * Initializes the delete sequence dialog with the correct content, and is responsible for
+     * deleting sequences via RGSequenceManager
+     * </summary>
+     */
+    public class RGDeleteSequence : MonoBehaviour
+    {
+        public TMP_Text sequenceNamePrefab; 
+        
+        public Button confirmButton;
+        
+        public Button cancelButton;
+
+        public string sequencePath;
+
+        /**
+         * <summary>
+         * Initializes the delete sequence dialog with the correct content
+         * </summary>
+         */
+        public void Start()
+        {
+            if (confirmButton != null)
+            {
+                confirmButton.onClick.AddListener(OnDelete);
+            }
+            else
+            {
+                Debug.LogError("RGDeleteSequence is missing its confirmButton");
+            }
+            
+            if (cancelButton != null)
+            {
+                cancelButton.onClick.AddListener(OnCancel);
+            }
+            else
+            {
+                Debug.LogError("RGDeleteSequence is missing its cancelButton");
+            }
+        }
+
+        /**
+         * <summary>
+         * Set the path and name of the Sequence the user is wanting to delete
+         * </summary>
+         * <param name="sequence">The Sequence Entry to delete</param>
+         */
+        public void Initialize(RGSequenceEntry sequence)
```

Comment:
If we can't delete project-based sequences from a Runtime build, maybe we can have this dialog show this? i.e. if we can see that the sequence is part of the build, disable the confirm button and have text that says something like "Sequences created as part of the Unity Editor can only be deleted when running in the Editor"?

Reply:
Ideally:

We should be able to grey out/disable the delete option for segments/sequences built into the project; when we are NOT in the editor.  Having a hover effect to explain that pre-built sequences/segments in the project can only be deleted in the Editor would give the user good feedback.

Reply:
@addisonbgross your change for disabling delete seems fine for this pass

in the future, we should consider whether we want to allow them to remove 'override' sequences in runtime builds

anything that isn't prebuilt into the project (anything loaded from the persistent data path) is deletable at runtime.. and in fact, may ONLY exist in that specific runtime and wouldn't be deletable from the editor.

that said.. do this as a follow up task because the changes i'm making for how runtime sequences are loaded will massively conflict with a change like this